addons/reputation/reputation.php

```diff
@@ -598,6 +599,31 @@ public function ap_all_options( $all_options ) {
 
 		return $all_options;
 	}
+
+	/**
+	 * Delete ap_reputation_events table datas,
+	 * since there is no icon set in the database for the reputation event,
+	 * and editing the table directly is not possible.
+	 */
+	public function ap_truncate_reputation_events_table() {
+		// Return if ap_reputation_events_contents_deleted exists.
+		if ( get_option( 'ap_reputation_events_contents_deleted' ) ) {
+			return;
+		}
+
+		global $wpdb;
+		// Delete the table contents of $wpdb->ap_reputation_events table.
+		$reputation_events_table = $wpdb->ap_reputation_events;
+		if ( $reputation_events_table ) {
+			$wpdb->query( "TRUNCATE TABLE {$reputation_events_table}" ); // phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared
```

Comment:
@mebishalnapit You cannot do this. You will have to write a proper db migration.

Reply:
@rahularyan Modified.